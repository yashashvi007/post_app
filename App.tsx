import { Text, View } from 'react-native'
import React, { Component } from 'react'
import Index from './src/routes';


class App extends Component {
 render(): React.ReactNode {
   return (
    <View>
      <Index/>
    </View>
   )
 }
}

export default App;