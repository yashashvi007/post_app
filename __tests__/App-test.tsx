import 'react-native';
import React from 'react';
import App from '../App';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';
// import { toMatchImageSnapshot } from 'jest-image-snapshot';
// expect.extend({ toMatchImageSnapshot });



test('App snapshot' , ()=> {
  const snap = renderer.create(
    <App/>
  ).toJSON();

  expect(snap).toMatchSnapshot();
  
})