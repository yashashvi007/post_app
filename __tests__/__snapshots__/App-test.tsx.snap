// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshot 1`] = `
<View>
  <View>
    <View>
      <View />
      <View />
      <View />
    </View>
  </View>
</View>
`;
